AsBuiltReport.Microsoft.AD.json

```diff
@@ -1,9 +1,9 @@
 {
     "Report": {
         "Name": "Microsoft AD As Built Report",
-        "Version": "1.0",
+        "Version": "0.2.0",
```

Comment:
This relates to the version of the document/report which is generated, not the ABR module.

Reply:
that's the drawback of using search&replace haha